### PR TITLE
Revert "Revert "Add images config map""

### PR DESCRIPTION
**Pull request by Danil-Grigorev**
Reverts openshift/cluster-cloud-controller-manager-operator#30

### CODE REVIEW

File: manifests/0000_26_cloud-controller-manager-operator_11_deployment.yaml [hunk 1]
```diff
@@ -49,3 +53,8 @@ spec:
         operator: "Exists"
         effect: "NoExecute"
         tolerationSeconds: 120
+      volumes:
+      - name: images
+        configMap:
+          defaultMode: 420
```
Comment (elmiko) [inline manifests/0000_26_cloud-controller-manager-operator_11_deployment.yaml:59]: is this the proper mode?

`chmod 420` produces `-r---w----` which doesn't look right to me.

File: hack/example-images.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "cloudControllerManagerAWS": "registry.svc.ci.openshift.org/openshift:aws-cloud-controller-manager",
+    "cloudControllerManagerOpenStack": "registry.svc.ci.openshift.org/openshift:openstack-cloud-controller-manager"
```
Comment (elmiko) [inline hack/example-images.json:3]: we should probably fix these up before committing. no reason to use the old url

File: manifests/0000_26_cloud-controller-manager-operator_01_images.configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  name: cloud-controller-manager-images
+  namespace: openshift-cloud-controller-manager-operator
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+data:
+  images.json: >
+    {
+      "cloudControllerManagerAWS": "registry.svc.ci.openshift.org/openshift:aws-cloud-controller-manager",
+      "cloudControllerManagerOpenStack": "registry.svc.ci.openshift.org/openshift:openstack-cloud-controller-manager"
```
Comment (elmiko) [inline manifests/0000_26_cloud-controller-manager-operator_01_images.configmap.yaml:13]: same here

File: manifests/image-references [hunk 4]
```diff
@@ -6,3 +6,11 @@ spec:
     from:
       kind: DockerImage
       name: registry.svc.ci.openshift.org/openshift:cluster-cloud-controller-manager-operator
+  - name: aws-cloud-controller-manager
+    from:
+      kind: DockerImage
+      name: registry.svc.ci.openshift.org/openshift:aws-cloud-controller-manager
```
Comment (elmiko) [inline manifests/image-references:12]: here too

**Review by elmiko — CHANGES_REQUESTED** [review]
@Danil-Grigorev this looks fine to me, but i think we should update the image urls now since we are adding them

File: hack/example-images.json (hunk 2 above)
Comment (Danil-Grigorev) [reply hack/example-images.json:3]: Right, let me adjust that. CI is now passing

File: manifests/image-references (hunk 4 above)
Comment (Danil-Grigorev) [reply manifests/image-references]: Fixed

**Review by elmiko — APPROVED** [review]
thanks Danil!
/approve

**Pull request merged**